### PR TITLE
Chore: Upgrade gix components

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use new functionality in ProgressBar.

`value` is optional and we can use the `testId` to select the progress bar and build a PO based on it. Specially useful if there were two progress bars within one component.

# Changes

* Upgrade gix-components

# Tests

Tests still pass.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/package-lock.json
Comment (dskloetd) [inline frontend/package-lock.json:1]: How is this different from https://github.com/dfinity/nns-dapp/pull/3348/files?
Do you just need to update your branch?

Comment (dskloetd) [reply frontend/package-lock.json:1]: I don't think it does. Look at the diff. Both have version `"version": "2.7.0-next-2023-09-21.2"`.
The diffs of this PR and the one I linked are actually identical.

Comment (dskloetd) [reply frontend/package-lock.json:1]: I'm approving because I'm going out for lunch but make sure you have the right version before merging.

**Review by dskloetd — APPROVED** [review]

File: frontend/package-lock.json
Comment (lmuntaner) [reply frontend/package-lock.json:1]: Oh, I deployed the wrong branch!

**Comment by lmuntaner** [comment]
Wrong branch.

**Pull request closed without merging**